src/main/java/com/attica/athens/domain/agora/api/AgoraOpenController.java

```diff
@@ -41,4 +44,24 @@ public ResponseEntity<ApiResponse<?>> getAgoraByKeyword(
 
         return ResponseEntity.ok(ApiUtil.success(response));
     }
+
+    @GetMapping("/{agora-id}/title")
+    public ResponseEntity<ApiResponse<?>> getAgoraTitle(
+            @PathVariable("agora-id") Long agoraId
+    ) {
+        AgoraTitleResponse agoraTitle = agoraService.getAgoraTitle(agoraId);
+
+        return ResponseEntity.ok(ApiUtil.success(agoraTitle));
+    }
+
+    @GetMapping("/agoraIdList")
+    public ResponseEntity<ApiResponse<?>> getAgoraIdList() {
+
+        AgoraIdResponse agoraIdList = agoraService.getAgoraIdList();
```

Comment:
p2: 변수던 url 경로이던 이름에 특정 타입이나 구현 부분을 포함하면 좋지 않다고 생각해요.
후에 구현이 충분히 바뀔 수 있으므로 유연성 측면에서 ids로 수정하는 것 제안드려요 !
```suggestion
    @GetMapping("/ids")
    public ResponseEntity<ApiResponse<?>> getAgoraIds() {

        AgoraIdResponse agoraIds = agoraService.getAgoraIds();
```

Reply:
그렇군요 ! 팁 감사합니다 :)